[2025-08-26T13:31:08.980+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-26T13:29:54.623885+00:00 [queued]>
[2025-08-26T13:31:08.991+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-26T13:29:54.623885+00:00 [queued]>
[2025-08-26T13:31:08.992+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-26T13:31:09.006+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): cleanup_temp_files> on 2025-08-26 13:29:54.623885+00:00
[2025-08-26T13:31:09.012+0000] {standard_task_runner.py:60} INFO - Started process 137 to run task
[2025-08-26T13:31:09.014+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mamda_monthly_etl_pipeline', 'cleanup_temp_files', 'manual__2025-08-26T13:29:54.623885+00:00', '--job-id', '333', '--raw', '--subdir', 'DAGS_FOLDER/mamda_monthly_etl.py', '--cfg-path', '/tmp/tmpy1rf7mwi']
[2025-08-26T13:31:09.016+0000] {standard_task_runner.py:88} INFO - Job 333: Subtask cleanup_temp_files
[2025-08-26T13:31:09.062+0000] {task_command.py:423} INFO - Running <TaskInstance: mamda_monthly_etl_pipeline.cleanup_temp_files manual__2025-08-26T13:29:54.623885+00:00 [running]> on host d06030c40481
[2025-08-26T13:31:09.138+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mamda_etl' AIRFLOW_CTX_DAG_ID='mamda_monthly_etl_pipeline' AIRFLOW_CTX_TASK_ID='cleanup_temp_files' AIRFLOW_CTX_EXECUTION_DATE='2025-08-26T13:29:54.623885+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-26T13:29:54.623885+00:00'
[2025-08-26T13:31:09.139+0000] {mamda_monthly_etl.py:242} INFO - Cleanup finished
[2025-08-26T13:31:09.140+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-08-26T13:31:09.149+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=mamda_monthly_etl_pipeline, task_id=cleanup_temp_files, execution_date=20250826T132954, start_date=20250826T133108, end_date=20250826T133109
[2025-08-26T13:31:09.188+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-26T13:31:09.203+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
